src/generators.jl

```diff
@@ -257,7 +258,8 @@ This enables us to generate a Havel Hakimi graph, which can
 be useful.
 """
 function _havel_hakimi(degs::Vector{Int}, store::Bool, ei::Vector{Int}, ej::Vector{Int})
-    q = PriorityQueue(Int,Int,Base.Order.Reverse)
+    # q = PriorityQueue(Int,Int,Base.Order.Reverse)
```

Comment:
Delete this old line

Reply:
Fixed here too 2488e36adc7a947be8cba864c5d175c95263bf12